cmd/auth/main.go

```diff
@@ -129,6 +139,11 @@ func loadConfig() config {
 		SSLRootCert: mainflux.Env(envDBSSLRootCert, defDBSSLRootCert),
 	}
 
+	kc := auth.KetoConfig{
+		WritePort: mainflux.Env(envKetoWritePort, defKetoWritePort),
+		ReadPort:  mainflux.Env(envKetoReadPort, defKetoReadPort),
```

Comment:
Write and Read ports are only used in `main`, so there is no need to have them in the KetoConfig. We can completely remove the KetConfig struct if we pass `Checker` and `Writer` to the fabric method (`NewPolicyCommunicator`).